ansible/server-state-playbooks/www-dev/www-dev.yml

```diff
@@ -38,7 +58,17 @@
       jekyll_build_git_repo: "https://github.com/openmicroscopy/www.openmicroscopy.org"
       jekyll_build_force_rebuild: True
 
-    - role: openmicroscopy.nginx
+    - role: openmicroscopy.nginx_proxy
+      tags: nginxconf
+      nginx_proxy_worker_processes: "{{ ((ansible_processor_count * ansible_processor_cores) / 2) |round|int }}"
+      nginx_proxy_ssl: True
+      nginx_proxy_ssl_certificate: "{{ nginx_ssl_files_path }}/{{ nginx_ssl_cert_filename }}"
+      nginx_proxy_ssl_certificate_key: "{{ nginx_ssl_files_path }}/{{ nginx_ssl_key_filename }}"
+      nginx_proxy_http2: True
+      nginx_proxy_force_ssl: False
+      nginx_proxy_hsts_age: 31536000
```

Comment:
Do you definitely want HSTS? `Testing HTTP Strict Transport Security with Care` https://www.nginx.com/blog/http-strict-transport-security-hsts-and-nginx/